packages/react-native/React/CoreModules/RCTDevLoadingView.mm

```diff
@@ -122,8 +122,8 @@ - (void)showMessage:(NSString *)message color:(RCTUIColor *)color backgroundColo
     self->_showDate = [NSDate date];
 
     if (!self->_window && !RCTRunningInTestEnvironment()) {
+      RCTUIWindow *window = RCTKeyWindow(); // [macOS]
```

Comment:
nit: I don't see us using `window` in the macOS block. Do we need this?

Reply:
Ah yeah, latest iteration doesn't. Removing.